app/questions/tests/test_reminder_email.py

```diff
@@ -0,0 +1,75 @@
+from django.utils import timezone
```

Comment:
generally these tests are really nice, great job picking up the testing so quickly

Reply:
thanks, testing can be fun ...